indico/modules/categories/client/js/components/CalendarLegend.jsx

```diff
@@ -0,0 +1,90 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import './CalendarLegend.module.scss';
```

Comment:
move imports with side effects to the end of the import list (they cannot be auto-sorted)

Reply:
Fixed